traits-stubs/traits_stubs_tests/test_all.py

```diff
@@ -0,0 +1,39 @@
+# (C) Copyright 2005-2020 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+
+
+from pathlib import Path
+from unittest import TestCase
+
+import pkg_resources
+
+from traits_stubs_tests.util import MypyAssertions
+
+examples_dir = Path(
+    pkg_resources.resource_filename('traits_stubs_tests', 'examples'))
+__unittest = True
```

Comment:
What is this line for? Is it necessary?

Reply:
This isn't necessary, it was for debugging, was trying to hide the long trackbacks. Not sure if it worked, but removing it anyway.